src/Angor/Client/Pages/Invest.razor

```diff
@@ -669,7 +663,7 @@ else
 
     private void UpdateStagesBreakdown(ChangeEventArgs e)
     {
-        if (decimal.TryParse(e.Value.ToString(), out decimal amountBtc))
+         if (decimal.TryParse(e.Value.ToString().Replace(",", "."), out decimal amountBtc))
```

Comment:
this seems totally incorrect, the browser culture should handle that.
In the Spanish language to represent decimals you use `,` not `.` so I think the paring should do that for you.